GEECS-PythonAPI/geecs_python_api/controls/interface/geecs_database.py

```diff
@@ -100,6 +105,10 @@ def _close_db(db, db_cursor):
     @staticmethod
     def collect_exp_info(exp_name: str = 'Undulator')\
             -> dict[str, Union[ExpDict, dict[str, Path], Path, int]]:
+        print(GeecsDatabase.name, "current name")
+        if GeecsDatabase.name is None:
+            raise AttributeError("Geecs Database not set properly")
+
         db = GeecsDatabase._get_db()
         db_cursor = db.cursor(dictionary=True)
 
```

Comment:
looks like print is widely used in geecs_database, so i guess ok to be consistent here for now.